woocommerce/woocommerce-android
# Add toolbar for CreateShippingLabelFragment

Part of #10570 

### Description

This PR introduces individual toolbars for CreateShippingLabelFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Order detail screen of an Order that has a shipping label (PbTz5e-11v-p2) (Order list -> Order detail -> Create a shipping label CTA) screen and ensure the toolbar and its action work as expected on both phone and tablet.

### Images/gif

<img width="244" alt="Screenshot 2024-02-12 at 6 59 37 PM" src="https://github.com/woocommerce/woocommerce-android/assets/1331230/c910dbdd-f525-446b-8d30-88439983b317">
